api/axis.py

```diff
@@ -547,6 +547,7 @@ def catch_up_from(self, start, cutoff, progress):
             count += 1
             if count % self.batch_size == 0:
                 self._db.commit()
+        progress.achievements = "Modified titles: %d." % count
```

Comment:
This is a little stilted but it's very easy to localize (no worries about plurals) and this isn't patron-facing prose.